exchange-contract/README.md

```diff
@@ -1,79 +1,92 @@
+The *issuer* contract object maintains a balance sheet that captures ownership of assets of a particular type. The issuer contract allows the creator of the contract object to issue assets (that is, assign ownership of assets to a particular individual). However, once the initial issuance occurs, confidentiality of transactions and balances is maintained; in this case, even the creator of the issuer contract object is not granted the right to examine the quantity of assets owned by an individual after the initial issuance. Local chapters of the Blue Marble Players Association each
+create an issuer contract object to assign ownership of marbles to their members. Once the initial issuance is complppete, members can trade marbles, transfer ownership, or exchange different kinds of marbles in complete confidentiality. And, for those who trust the Blue Marble Players Association, those transactions can span local chapters.
 
-- The NFT smart contract, being public in nature, cannot be used to either store any confidential information (such as a decryption key to the asset), or execute a compute-logic that contains confidential information. 
+While the issuer contract object supports simple ownership transfer, more complex multi-party exchanges are managed through additional contracts. For example, an exchange contract mediates a fair exchange of different kinds of marbles (e.g. Alice trades her red marbles for Bob's blue marbles). More complex exchanges like a blind auction can be implemented as well.
 
-In this repo, we propose systems and protocols that address the above inefficiencies so that description/enforcement of asset-usage policies can be implemented as part of the NFT smart contract itself, thereby widely enhancing the scope of what can be done with NFTs.
+## Contracts for Confidential, Non-Fungible Assets
 
-## Solution Overview
+One blue marble is generally indistinguishable from another; that is, blue marbles are considered "fungible". In contrast, each red sports car is a unique thing with a distinct title of ownership; that is, red sports cards are considered "non-fungible". In order to represent non-fungible assets like a red sports car, the Exchange contract family defines a special kind of issuer, called a *token issuer*. Like the issuers for fungible tokens, the token issuer has an asset type and vetting organization. A token issuer contract object "mints" a fixed number of *tokens* (another kind of contract object) that correspond to unique instances of the asset type. To make this more concrete, the asset type might refer to red sports cars (a generic kind of car), the token issuer is like a manufacturer that creates specific instances (tokens) of the red sports car. Each token represents a specific red sports car.
```

Comment:
I like this analogy, though given the next paragraph it strikes me that a token is more about access than ownership. I wonder if you could expand on this analogy just a bit and equate a token with a car key? Anyone with a car key that has been issued for a specific car can operate that car. Is this the right way to think about the analogy?

Reply:
uggh... really????? :)

Reply:
my question stands :)